ios/RNOneginiSdk.swift

```diff
@@ -21,15 +21,34 @@ enum OneWelcomeBridgeEvents: String {
 @objc(RNOneginiSdk)
 class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
     var bridgeConnector: BridgeConnector
-    private var userClient: ONGUserClient {
-        return ONGClient.sharedInstance().userClient
+    
+    private lazy var sharedClient = ClientBuilder().build()
+    
+    private var userClientONG: ONGUserClient {
+        // We use this computer property to make sure the SDK is built before we use it. This can be removed once we fully switch to the swift api.
+        let client = sharedClient
```

Comment:
No need to create unneeded variable/constant. `sharedClient` or `_ = sharedClient` should be fine to meet your expectations.